client/blocks/plan-storage/docs/example.jsx

```diff
@@ -2,86 +2,100 @@
  * External dependencies
  */
 import React from 'react';
-import PureRenderMixin from 'react-pure-render/mixin';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
 
 /**
  * Internal dependencies
  */
 import PlanStorage from '../index';
 import PlanStorageBar from '../bar';
-import sitesList from 'lib/sites-list';
 import {
 	PLAN_BUSINESS,
 	PLAN_PREMIUM,
 	PLAN_PERSONAL,
 	PLAN_FREE,
 } from 'lib/plans/constants';
+import { getCurrentUser } from 'state/current-user/selectors';
+import { getSite, getSiteSlug } from 'state/sites/selectors';
 
-const sites = sitesList();
+const PlanStorageExample = ( { site, siteId, siteSlug } ) => {
+	const mediaStorage = {
+		red: {
+			storage_used_bytes: 11362335981,
+			max_storage_bytes: 13958643712
+		},
+		yellow: {
+			storage_used_bytes: 1971389988,
+			max_storage_bytes: 3221225472
+		},
+		green: {
+			storage_used_bytes: 167503724,
+			max_storage_bytes: 3221225472
+		}
+	};
 
-export default React.createClass( {
+	if ( ! site ) {
```

Comment:
Ah, good call :+1: Fixed in https://github.com/Automattic/wp-calypso/pull/14114/commits/61ae6e5f725084cbaa47525951b74aa2b91ceaf4.